packages/store/src/decorators/select.ts

```diff
@@ -19,7 +20,7 @@ export class SelectFactory {
 /**
  * Decorator for selecting a slice of state from the store.
  */
-export function Select(selectorOrFeature?, ...paths: string[]) {
+export function Select(selectorOrFeature?: any | string[], ...args: any[]) {
```

Comment:
This change would unfortunately be a breaking change for the framework. This is not an option.